static/app/stores/tagStore.tsx

```diff
@@ -86,6 +86,7 @@ const storeConfig: TagStoreDefinition = {
           IssueCategory.ERROR,
           IssueCategory.PERFORMANCE,
           ...(org.features.includes('issue-platform') ? [IssueCategory.PROFILE] : []),
+          ...(org.features.includes('crons-issue-platform') ? [IssueCategory.CRON] : []),
```

Comment:
@rjo100 aren't we deleting this?

Reply:
I picked a feature flag that made sense, let me know if there's another to use

Reply:
oh i'm about to delete this yeah. just do `issue-platform`